### PR TITLE
Added Create Category functionality

### PR DESCRIPTION
**Pull request by AyushTripathi-sketch**
Solved issue #103 

Category List Screen Before: 

1st

Category Create Screen:

2nd

Category Screen After:

3rd

### CODE REVIEW

**Comment by roopeshsn** [comment]
Hi, @AyushTripathi-sketch Could you add a screen-recorded video of your work? Also, could you summarize what changes you have done? Have you formatted the code using prettier?

**Comment by AyushTripathi-sketch** [comment]
Yes, I can scree record my work. I have created a screen for creating category and updated the category action and reducers for deleting, updating and creating category. In the backend I have made a category controller for different for the same actions and added a admin/createcategory route. I have also formatted the code using Prettier..
